### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by classifiedos**
At this point, basic operations (add, delete, create) are working for the LinkedList.   As this isn't serious production code, let's merge to master so that we each have flexibility to work from the most recent version of the code.  What do you think?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.class
```
Comment (declassified) [inline .gitignore:1]: Wow we are pretty clever. If only lecture was this easy to .gitignore ...

Comment (classifiedos) [inline .gitignore:1]: Very funny...

**Comment by declassified** [comment]
yeah ok

**Comment by classifiedos** [comment]
thanks

**Pull request merged**